src/corg/developer-environment-setup/installing-dart-and-related-tools.md

```diff
@@ -39,13 +39,13 @@ history:
 
 ---
 
-See [https://www.dartlang.org/tools/download.html](https://www.dartlang.org/tools/download.html)
+See <https://www.dartlang.org/tools/download.html>
 
 ## Installing Dart
 
 ### Linux
 
-{{#> panel type='code' heading='Manual install of Dart'}}
```

Comment:
Because the paragraph below, for OS X, does not have panel. This different setting gives a different UI in the page.

Now, I reverted the change of this line, and added a panel for the OS X section.